### DISCUSSION TITLE
Django like Routes in Fastapi

### DISCUSSION THREAD

**Discussion by moinologics** [fastapi/fastapi#8769] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
urlpatterns = [
    path('path/1', funtion_name),
    path('path/2/', another_function)
]
```

### Description

I need clear routing

Also is there a way to make a separate route file?

### Operating System

Linux

### Operating System Details

Ubuntu 20.04

### FastAPI Version

0.61

### Python Version

3.8

### Additional Context

There must be some clear routing machenism like in laravel & django framework.

I like FastApi very much but you selected flask like Routes, which I think is very bad.

**Comment by panla** [comment, marked as answer]
```python
from fastapi import FastAPI

app = FastAPI()

def index():
    return {'success': True}


app.add_api_route(path='/index', endpoint=func)
```
